### PR TITLE
Bugfix) if emmc is not ready try again in 1sec

### PR DESCRIPTION
**Pull request by jontore**

### CODE REVIEW

File: src/components/upgrader/boxfishUpgrader.ts [hunk 1]
```diff
@@ -444,6 +445,11 @@ export default class BoxfishUpgrader extends EventEmitter implements IDeviceUpgr
       const response = await this._cameraManager.getState();
 
       this._logger.info(`Upgrade status ${JSON.stringify(response)}`);
+      if (response.status === 10) {
+        // EMMC is not ready lets wait and try again
+        await new Promise(resolve => setTimeout(resolve, 1000));
```
Comment (brikendr) [inline src/components/upgrader/boxfishUpgrader.ts]: Why is this necessary?

Comment (jontore) [reply src/components/upgrader/boxfishUpgrader.ts]: To give the EMMC time to come up (status 10, means it is not ready yet)

Comment (brikendr) [reply src/components/upgrader/boxfishUpgrader.ts]: Alright. Nice
